### PR TITLE
Fix for ref property in RPC WSDL for 6.x

### PR DESCRIPTION
**Pull request by nalakas**
Fix for ref property

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/soaptorest/WSDL11SOAPOperationExtractor.java [hunk 1]
```diff
@@ -623,8 +623,15 @@ private WSDLSOAPOperation getSOAPOperation(BindingOperation bindingOperation) th
                                     ModelImpl model = new ModelImpl();
                                     model.setType(ObjectProperty.TYPE);
                                     model.setName(message.getQName().getLocalPart());
-                                    model.addProperty(part.getName(),
-                                            getPropertyFromDataType(part.getTypeName().getLocalPart()));
+                                    if (getPropertyFromDataType(part.getTypeName().getLocalPart()) instanceof RefProperty) {
+                                        RefProperty property = (RefProperty) getPropertyFromDataType(part.getTypeName()
+                                                .getLocalPart());
+                                        property.set$ref("#/definitions/" + part.getTypeName().getLocalPart());
```
Comment (uvindra) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/soaptorest/WSDL11SOAPOperationExtractor.java]: Lets use a constant here

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/soaptorest/WSDL11SOAPOperationExtractor.java [hunk 2]
```diff
@@ -670,8 +677,15 @@ private WSDLSOAPOperation getSOAPOperation(BindingOperation bindingOperation) th
...
```
Comment (uvindra) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/soaptorest/WSDL11SOAPOperationExtractor.java]: Lets use a constant here

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/soaptorest/WSDL11SOAPOperationExtractor.java (hunk 1 above)
Comment (nalakas) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/soaptorest/WSDL11SOAPOperationExtractor.java]: fixed in 
1b14dfb

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/soaptorest/WSDL11SOAPOperationExtractor.java (hunk 2 above)
Comment (nalakas) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/soaptorest/WSDL11SOAPOperationExtractor.java]: fixed in 
1b14dfb

**Pull request merged**